client/src/pages/demo/components/index.tsx

```diff
@@ -158,6 +159,17 @@ const DemoComponent: React.FunctionComponent = () => {
               the event of the page
             </div>
           </div>
+          <div className="p-4 border-b border-black space-y-4">
+            <Searchbar onSearchEvent="string" />
+            <div className="mt-1.5">
+              <h1 className="font-semibold">Props: </h1>
+              <div className="bg-gray-300 p-1.5">
+                <p>onSearchEvent = (string)[""] ex. onSearchEvent="string" </p>
+                <p>height = (string)[auto] ex. height="50px"</p>
+                <p>width = (string)[auto] ex. width="100px"</p>
+              </div>
+            </div>
+          </div>
```

Comment:
- Can you put this outside the div of the pagination component.
- This is to emphasize the component you created